extensions/amp-story-360/0.1/amp-story-360.js

```diff
@@ -411,6 +438,19 @@ export class AmpStory360 extends AMP.BaseElement {
     }
   }
 
+  /** @private */
+  restoreOrLoseGlContext_() {
+    if (this.renderer_) {
```

Comment:
Nit: early return to save one level of indentation

Too bad we can't use `isReady_` btw : (

Reply:
That feels more semantically logical but what we need here is the renderer.

isReady_ is only set to true after the renderer is set up along with the image.